README.md

```diff
@@ -1 +1,16 @@
-# API
+# TheCookieMC API
+Here you can find the documentation for TheCookieMC's API! All requests are sent through `api.thecookiemc.net`. The system currently only works with player UUIDs, not usernames. As the server is under active development, some API requests may be removed or changed without prior notice. 

```

Comment:
The welcome shouldn't be in the same place as api information.

Reply:
The "without piror notice" isn't accurate - notice will be given in the discussions tab of github.

Reply:
Updated. Not sure what you mean by "The welcome shouldn't be in the same place as api information."